src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/Namespace/NewAzureEventHubNamespace.cs

```diff
@@ -92,15 +92,34 @@ public class NewAzureEventHubNamespace : AzureEventHubsCmdletBase
         public Hashtable Tag { get; set; }
 
         /// <summary>
+        /// Indicates whether AutoInflate is enabled.
+        /// </summary>
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Indicates whether AutoInflate is enabled")]
+        public bool? IsAutoInflateEnabled { get; set; }
+
+        /// <summary>
+        /// Upper limit of throughput units when AutoInflate is enabled.
+        /// </summary>
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Upper limit of throughput units when AutoInflate is enabled, vaule should be within 0 to 20 throughput units.")]
+        public int? MaximumThroughputUnits { get; set; }
```

Comment:
@v-Ajnava please use `ValidateSet` if you're going to limit the value to be between 0-20

Also, there is a typo in the above HelpMessage: _vaule_ should be _value_